### PR TITLE
Add Collections page content

### PR DESCRIPTION
**Pull request by jgagne**
## What this PR does:

- Adds Collections data
- Adds Jekyll logic
- Updates style

### Content Preview

- https://deploy-preview-575--thegymcms.netlify.app/static/collections/

### Design Preview

gym-collections-landing-page-comp

### CODE REVIEW

File: hub-pages/collections/collections.html [hunk 1]
```diff
@@ -21,23 +21,24 @@ <h1>Collections</h1>
...
           <div class="collection--cta">
-            <a class="gym-button gym-button-secondary" href="placeholder-url">
+            <a class="gym-button gym-button-secondary" href="{{ hub_page.url }}">
               <b>Explore</b>
             </a>
           </div>
         </div>
       </li>
-    <!-- placeholder endfor -->
+    {%- endfor -%}
```
Comment (andrewpmiller) [inline hub-pages/collections/collections.html:41]: @jgagne — This is some seriously slick work!

Comment (jgagne) [reply hub-pages/collections/collections.html:41]: @andrewpmiller — Thank you! All from a slick sorcerer. 🧙

**Review by rediris — APPROVED** [review]
Loos good to me, and nice work with the YAML magic

**Review by andrewpmiller — APPROVED** [review]
Magnifique!

**Comment by andrewpmiller** [comment]
giphy (1)

**Pull request merged**